packages/amplify-category-auth/provider-utils/awscloudformation/triggers/PostConfirmation/cloudformation-templates/PostConfirmation.json.ejs

```diff
@@ -196,6 +196,10 @@
                 "Ref": "LambdaExecutionRole"
             }
         },
+        "LambdaExecutionRoleArn": {
+            "Value": {"Fn::GetAtt": ["LambdaExecutionRole", "Arn"]}
```

Comment:
Could you reformat like the expression above, into multiple lines with proper ident?